scripts/validate-dashboards.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -eu
+
+# List of files to exclude
+excludes=( "values.schema.json" "home.json" )
+excludes_string="$(IFS="|"; echo "${excludes[*]}")"
+
+# Find valid team names
+github_repo="https://github.com/giantswarm/github.git"
+github_repo_directory="$(mktemp -d -t validate-dashboards.XXXXXXXXXX)"
+trap 'rm -rf "$github_repo_directory"' EXIT
+git clone --depth 1 "$github_repo" "$github_repo_directory"
+pushd "$github_repo_directory" > /dev/null || exit 1
+mapfile teams < <(find repositories -name '*.yaml' -printf '%f\n' | cut -d. -f1)
+popd > /dev/null || exit 1
+echo "Found ${#teams[@]} teams:" ${teams[*]}
+
+# Check all dashboards
+failures=0
+log_length=19
+exec 5< <(find helm/dashboards/ -name "*.json" | grep -vE "$excludes_string")
```

Comment:
This is nice because it allows you to write the loop command on top of the loop. But the syntax is even wierder than the classic one, so I think it deserves some comments so people who are less at ease with bash won't run away :rofl: 

The classic one I think about:
```
while read ; do
  work on found files
done < <(find whatever)
```
But I may even be wrong when thinking this is what your code does :sweat_smile:

Reply:
That's what my code does, it's just a fancy way to keep the content being looped on before the loop. But I can switch this to the traditional way

Reply:
Changed this to 

```
find whatever | while read; do
  work
done
```

WDYT ?

Reply:
If that works the same, I think it's much easier to read! :+1: